src/test/java/svnserver/ldap/EmbeddedDirectoryServer.java

```diff
@@ -47,8 +76,56 @@ private EmbeddedDirectoryServer(@NotNull String dn, @NotNull URL ldifStream) thr
   }
 
   @NotNull
-  public static EmbeddedDirectoryServer create() throws Exception {
-    return new EmbeddedDirectoryServer("dc=example,dc=com", EmbeddedDirectoryServer.class.getResource("ldap.ldif"));
+  private static KeyManager[] createKeyManagers(@NotNull File certFile, @NotNull File keyFile) throws Exception {
+    final X509Certificate certificate = LdapUserDB.loadCertificate(certFile);
+
+    final PrivateKey key = loadKey(keyFile);
+
+    final char[] keyPassword = "12345".toCharArray();
+    final KeyStore keyStore = assembleKeyStore(certificate, key, keyPassword);
+
+    final KeyManagerFactory keyManagerFactory = KeyManagerFactory.getInstance(KeyManagerFactory.getDefaultAlgorithm());
+    keyManagerFactory.init(keyStore, keyPassword);
+
+    return keyManagerFactory.getKeyManagers();
+  }
+
+  @NotNull
+  private static KeyStore assembleKeyStore(@NotNull Certificate certificate, @NotNull PrivateKey key, @NotNull char[] keyPassword) throws KeyStoreException, IOException, NoSuchAlgorithmException, CertificateException {
+    final KeyStore keyStore = KeyStore.getInstance(KeyStore.getDefaultType());
+    keyStore.load(null);
+
+    keyStore.setKeyEntry("key", key, keyPassword, new Certificate[]{certificate});
+    return keyStore;
+  }
+
+  @NotNull
+  private static PrivateKey loadKey(@NotNull File keyFile) throws Exception {
```

Comment:
I failed to find a way to load private key via standard Java API, so using `org.bouncycastle.openssl` here and then converting to Java standard `PrivateKey`.